src/openforms/template/backends/sandboxed_django.py

```diff
@@ -43,7 +43,7 @@ def get_templatetag_libraries(self, custom_libraries: dict) -> dict:
         This prevents user-content from loading and using the libraries that are
         private API to Open Forms.
         """
-        return {}
+        return custom_libraries
```

Comment:
I don't think so? Since it is initialised with `backend = SandboxedDjangoTemplates({})` and so the params are `{}` which then results in `libraries` to be empty?

Reply:
I've added an explicit test to confirm this - it works because we set the default for `libraries` in the init params to empty dict and don't specify this anywhere else & because the method override basically removes Django's template tag discovery. At least now there is an explicit test :)

I'm not sure why this change was needed though, because we don't expose them via `libraries`, but via `builtins` instead :thinking: